__tests__/app.test.js

```diff
@@ -38,3 +38,57 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with an object of the article with the correct article id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(typeof article).toBe("object");
+        expect(typeof article.article_id).toBe("number");
```

Comment:
We haven't actually checked that it's the right article - just that its a number but not which one!